Detections/SecurityEvent/SecurityEventLogCleared.yaml

```diff
@@ -17,9 +18,9 @@ relevantTechniques:
   - T1107
 query: |
```

Comment:
Not required, but we like to have an empty line between "query:  |" and the start of the query.